torch/csrc/jit/ir/ir.cpp

```diff
@@ -1144,40 +1144,19 @@ Operation Node::getOperation() const {
 }
 
 bool Node::isNondeterministic() const {
-  static const OperatorSet nondeterministic_ops = {
-      "aten::dropout(Tensor input, float p, bool train) -> Tensor",
-      "aten::_fused_dropout(Tensor self, float p, Generator? generator) -> (Tensor, Tensor)",
-      "aten::_standard_gamma(Tensor self, Generator? generator) -> Tensor",
-      "aten::bernoulli(Tensor self, *, Generator? generator) -> Tensor",
-      "aten::bernoulli(Tensor self, float p, *, Generator? generator) -> Tensor",
-      "aten::multinomial(Tensor self, int num_samples, bool replacement, *, Generator? generator) -> Tensor",
-      "aten::native_dropout(Tensor input, float p, bool? train) -> (Tensor, Tensor)",
-      "aten::normal(Tensor mean, Tensor std, *, Generator? generator) -> Tensor",
-      "aten::normal(float mean, Tensor std, *, Generator? generator) -> Tensor",
```

Comment:
could you also add a test for these ops to make sure they are all getting marked as nondeterministic?

To make it easier (i.e. to avoid having to figure out how to write a valid test using each of the ops) you could use opinfo tests and filter the set of ops to this list, example below:

https://github.com/pytorch/pytorch/blob/master/test/test_jit_fuser_te.py#L2593

You can do it as a separate PR if it's easier